libexec/scoop-import.ps1

```diff
@@ -0,0 +1,44 @@
+# Usage: scoop import <filepath/filepaths> [options]
+# Summary: Import exported list!
+# Help: The basic way to install exported list:
+#      scoop import exportedfile.txt
+# It also supports for multiple file:
+#      scoop import file1.txt file2.txt
+
+function createAppsList($filePaths){
+    if($filePaths){
+        foreach ($file in $filePaths){
+            $ext = [IO.Path]::GetExtension($file)
+            if($ext -eq ".txt")
+            {
+                foreach ($line in Get-Content $file)
+                {
+                    if($line -ne "") #Checks empty line!
+                    {
+                        $appslist += $line.Split()[0] + " "
```

Comment:
What if we have duplicate packages on the list or dependency of some package and the package itself?

Reply:
Duplicate packages are also going to into list but, because of the nature ``scoop install`` they are not installed. But you are right, duplicates are trying to install by this script. I will look into it. Thx for your comments! 🥂